tests/sweep_framework/sweeps/conv2d/short/conv2d_short_sweep.py

```diff
@@ -20,8 +20,6 @@
             # Contains following params
             # [batch_size, output_channels, input_channels, input_height, input_width, kernel_height, kernel_width, stride_x, stride_y, pad_x, pad_y, groups, bias, dilation]
             [1, 32, 1, 28, 28, 3, 3, 1, 1, 0, 0, 1, True, 1],
-            [1, 32, 1, 28, 28, 3, 3, 1, 1, 0, 0, 1, True, 1],
-            [1, 100, 100, 14, 14, 3, 3, 1, 1, 1, 1, 100, False, 1],
```

Comment:
Why remove this combination?

Reply:
Because they don't fail any more here I keep only params from short sweep that still fail